modules/integration/src/test/scala/scala/cli/integration/SipScalaTests.scala

```diff
@@ -122,6 +122,20 @@ class SipScalaTests extends ScalaCliSuite {
       }
     }
 
+  def checkHelp(binaryName: String): Unit = TestInputs.empty.fromRoot { root =>
+    val cliPath = os.Path(TestUtil.cliPath, os.pwd)
+    val ext = if (Properties.isWin) ".exe" else ""
+    val newCliPath = root / s"$binaryName$ext"
+    os.copy(cliPath, newCliPath)
+
+    for { helpOption <- Seq("help", "-help", "--help") } {
```

Comment:
Should be sufficient if this test will test only `help` command. For `-help` and `--help` it will be the same effect.

Reply:
I mean, we do already test for `version` along with `-version` and `--version` in `SipScalaTests`...
And `HelpTests` similarly already test for `help`, `-help` and `--help`...
So I see no reason to skip over that here, either (even if it's a little paranoid)